src/main/java/uk/gov/beis/els/service/TemplatePopulator.java

```diff
@@ -292,32 +294,36 @@ public Document getPopulatedDocument() {
     return template;
   }
 
-  public <T extends AnalyticsForm & SupplierNameForm> ProcessedEnergyLabelDocument asProcessedEnergyLabel(
+  public <T extends BaseForm & SupplierNameForm> ProcessedEnergyLabelDocument asProcessedEnergyLabel(
       ProductMetadata analyticsLabel, T form) {
-    String analyticsAction = String.format("%s - %s", form.getSupplierName(), form.getModelName());
+    EnergyLabelFormat labelFormat = getOutputFormatOrDefault(form);
+    String analyticsAction = String.format("%s - %s - %s", form.getSupplierName(), form.getModelName(), labelFormat);
     String title = String.format("%s - %s", analyticsLabel.getProductFileName(), analyticsAction);
```

Comment:
Nit: This means that the final filename ends up containing the image format (eg `Air conditioners - Some Supplier - Some Model - JPEG.jpg`) which is redundant when you have the file extension anyway. Could we rejig this so that the analytics action still contains the label format but it's not included in the filename?

Reply:
Ah fair point, will update